### DISCUSSION TITLE
Update issue!!

### DISCUSSION THREAD

**Discussion by 0xhunster** [projectdiscovery/naabu#318] [General]
When I going to update naabu, I saw this error!
![image](https://user-images.githubusercontent.com/46501627/167777745-73cd7fb0-eb98-4aae-bd95-e9795804054b.png)

**Comment by Mzack9999** [comment]
@0xhunster Thanks for opening this issue. You need to install `libpcap`:
```console
sudo apt-get update
sudo apt-get install libpcap-dev
```

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
